cl/sentinel/discovery.go

```diff
@@ -122,8 +132,13 @@ func (s *Sentinel) listenForPeers() {
 			continue
 		}
 
+		if err := sem.Acquire(s.ctx, 1); err != nil {
+			log.Error("Failed to acquire sem for opening peer connection", "err", err)
```

Comment:
- pls add log prefix like `[caplin]` or `[sentinel]`
- if err is `context.Cancel` then need `return/break`. and in this case - don't need write logs - because "graceful shutdown" it's not an "error flow".